### PR TITLE
Migrated from JavaScript to TypeScript : components/Footer/Footer.js

### PR DESCRIPTION
**Pull request by shukapurv**
**Fixes**

Fixes part of #93 by @shubhanshu02 

**Description**

**Checklist:**

-   [x] I have read the contribution guidelines in the repository.
-   [x] My branch is up-to-date with upstream branch.
-   [x] My pull request has a descriptive title (not a vague title like "Update package.json").
-   [x] My code follows the established code style of the repository.
-   [x] I have tried running the project locally and verified that there are no visible errors.

### CODE REVIEW

File: components/Footer/Footer.tsx [hunk 1]
```diff
@@ -24,22 +27,12 @@ export default function Footer() {
...
                     </div>
                     <div>
-                        <Text
-                            tag="h6"
-                            textSize="paragraph"
-                            textColor="#003e54c9"
-                            fontFamily="madetommy-light"
-                        >
+                        <Text {...TextProps}>
                             Hackalog is made with 🍕, ☕ and 💻
```
[the remark this answers is not in the document]
Comment (shukapurv) [reply components/Footer/Footer.tsx]: Actually ESLint was disabled so missed this warning btw i have fixed it.

File: pages/_app.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ import Head from "next/head"
 
 import { AuthProvider } from "../context/auth"
 import Header from "../components/Header/Header"
-import Footer from "../components/Footer/Footer"
+import Footer from "../components/Footer/Footer.tsx"
```
[the remark this answers is not in the document]
Comment (shukapurv) [reply pages/_app.js]: Reverted it

File: components/Footer/Footer.tsx [hunk 3]
```diff
@@ -24,22 +27,12 @@ export default function Footer() {
...
                     <div>
-                        <Text
-                            tag="h6"
-                            textSize="paragraph"
-                            textColor="#003e54c9"
-                            fontFamily="madetommy-light"
-                        >
+                        <Text {...TextProps}>
                             <Link href="/contributors">
                                 <a rel="noopener noreferrer">✨ CONTRIBUTORS</a>
```
[the remark this answers is not in the document]
Comment (shukapurv) [reply components/Footer/Footer.tsx]: Fixed it also

File: components/Footer/Footer.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (shubhanshu02) [reply components/Footer/Footer.tsx]: You forgot the coffee 😅?

Also do pull the latest changes from this repository to your branch for this PR.

[the remark this answers is not in the document]
Comment (shukapurv) [reply components/Footer/Footer.tsx]: oh my bad :sweat_smile: wrapped it also and pulled the latest changes.

[the remark this answers is not in the document]
Comment (shubhanshu02) [reply components/Footer/Footer.tsx]: No problem

**Comment by shubhanshu02** [comment]
Why are there changes in the `package-lock.json`? :thinking:

**Comment by shukapurv** [comment]
> Why are there changes in the `package-lock.json`?

I tried to figure out the reason, I guess there are some conflicts in package-lock.json. 
May you help me out, how to resolve it? @shubhanshu02

**Comment by shubhanshu02** [comment]
Can you confirm your npm version?

**Comment by shukapurv** [comment]
> Can you confirm your npm version?

Currently using 7.22.0

**Comment by shubhanshu02** [comment]
Ya just saw these lines in there:

```
<<<<<<< HEAD
       },
       "peerDependenciesMeta": {
         "supports-color": {
           "optional": true
         }
=======
>>>>>>> upstream/main
```
Try rebasing instead of merging changes. `git pull upstream main --rebase`. If the problem still persists, run `npm install` again.

**Review by shubhanshu02 — APPROVED** [review]

**Pull request merged**